HW5/KachanovskiAnton/task_3.py

```diff
@@ -0,0 +1,7 @@
+entered_number = int(input("Enter the number for which to calculate the factorial: "))
```

Comment:
good job
you can also take into account the cases when the entered number is negative or equal to zero